src/points/slice.test.ts

```diff
@@ -59,4 +61,12 @@ describe('pending points events', () => {
 
     expect(newState.pendingPointsEvents).toEqual([pendingSwapEvent])
   })
+
+  it('should set intro has been seen', () => {
+    const initialState = { introHasBeenDismissed: false } as unknown as State
```

Comment:
nit: use `initialState` rather than casting this object to the `State` type - will ensure that we get better type errors (e.g. if you change the name or type of this variable in the slice, this test probably won't complain until you run it)